hphp/runtime/ext/pdo_mysql/pdo_mysql.cpp

```diff
@@ -203,11 +203,16 @@ bool PDOMySqlConnection::create(const Array& options) {
 #ifdef CLIENT_MULTI_RESULTS
     |CLIENT_MULTI_RESULTS
 #endif
-#ifdef CLIENT_MULTI_STATEMENTS
-    |CLIENT_MULTI_STATEMENTS
-#endif
     ;
 
+  #ifdef CLIENT_MULTI_STATEMENTS
+    if (!options.empty()) {
```

Comment:
This logic is the inverse of what is intended.
The way you've written it here, it will default to no multi-statement, but enable multi-statements if you've provided any options.

This needs to be `if (options.empty())` instead.